doc/Learn/ThemeService/HowTo-UseThemeService.md

```diff
@@ -1,40 +1,29 @@
 ---
 uid: Uno.Extensions.ThemeService.Overview
 ---
-
 # How to use Theme Service
 
 This topic explains how to use the `ThemeService` for runtime theme switching and persisting user theme preferences.
 
-## Step-by-step
+## Automatic Registration with IoC
```

Comment:
How about using 'DI' instead of 'IoC'? In my head it feels more straightforward and what most developers expect when talking about dependency injection.

Reply:
Good point, I think both terms make sense for this + it is true that we use DI across other parts as well 👍